### DISCUSSION TITLE
What is wrong with the following decision table?

### DISCUSSION THREAD

**Discussion by antback** [gorules/zen#282] [Q&A] [answered]
<img width="1716" alt="image" src="https://github.com/user-attachments/assets/fa7d8699-2413-4aca-a7fb-82a254cb15b4">

You can see in the image that FlowA decision table receives 

```json
{
    "source": "a",
    "provider": "Australian Post"
}
```
And it sets the output to invalid. What am I missing?

Thanks.

**Comment by antback** [comment]
If I remove the third line, the output is null:

<img width="775" alt="image" src="https://github.com/user-attachments/assets/f438e03f-26b3-4391-a382-ffa1c26e2592">

```
{
  input: {
    provider: 'Australian Post',
    source: 'a',
  },
  output: null,
  traceData: null,
}
```

**Comment by stefan-gorules** [comment]
Would you be able to share graph JSON and version of the engine?

**Comment by antback** [comment]
Hi @stefan-gorules,

I'm testing directly in https://editor.gorules.io/. Here is the input json

```
{
    "source": "a",
    "provider": "Australian Post"
}
```

and the graph JSON file:

[jdm_graph.json](https://github.com/user-attachments/files/17995389/jdm_graph.json)

**Comment by stefan-gorules** [comment]
You need to remove inputField "case":

![CleanShot 2024-12-03 at 21 21 20@2x](https://github.com/user-attachments/assets/1ebb4cf2-ed8b-4c1f-8460-babb91f1472b)

After this, decision table should work as expected.

**Reply by antback** [reply]
Could you please share the graph JSON? I don't know what happens but it does not work ...

**Reply by antback** [reply]
There is something weird. When I open the graph JSON, I don't see `case` in the input field:

<img width="864" alt="image" src="https://github.com/user-attachments/assets/32210fbb-0f2b-4c68-b1e7-c7a4af615cce">

But looking the JSON file in a text editor, it is shown:

<img width="792" alt="image" src="https://github.com/user-attachments/assets/f38c2153-60ca-48e4-9284-14b08ff11535">

**Reply by ivanmiletic** [reply, marked as answer]
I believe there was a bug in that version of Editor in regards to input fields / output paths. We have fixed it but it is not yet released on editor.gorules.io, we will try to release it tomorrow.

It should be fixed on our BRMS / demo cloud.

**Reply by antback** [reply]
Thanks for your help @ivanmiletic. Running the editor with docker works.
